README.md

```diff
@@ -71,34 +73,38 @@ To run both the backend and frontend during development, use the following comma
 
 ```bash
 wishop/
-|-- src/
+|-- packages/
```

Comment:
I would think that this part of folder structure will change, or probably you could leave it to @pierratono to update it. I mean we agreed to use default nestJs and nextjs folder structure

Reply:
Hi @Nziranziza, sure: I hope he will also update this particular part in the folder structure PR. We can enforce it in his PR to include this particular part of the README update and keep the rest of it.

Let us know your review decision if you think the rest of changes introduced in here are relevant.